manticore/platforms/linux_syscall_stubs.py

```diff
@@ -1187,3 +1189,18 @@ def sys_wait4(self, upid, stat_addr, options, ru) -> int:
     def sys_waitid(self, which, upid, infop, options, ru) -> int:
         """ AUTOGENERATED UNIMPLEMENTED STUB """
         return self.simple_returns()
+
+    @staticmethod
+    def unimplemented_syscalls() -> Iterable[str]:
+        import inspect
+
+        return (
+            x[0].split("sys_", 1)[1]
+            for x in inspect.getmembers(SyscallStubs, predicate=inspect.isfunction)
+            if x[0].startswith("sys_")
+        )
+
+    @staticmethod
+    def print_unimplemented_syscalls() -> None:
+        for syscall in SyscallStubs.unimplemented_syscalls():
+            print(syscall)
```

Comment:
This is moved in #2491 to use a more reliable and flexible source of system call information